lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenProjectCache.java

```diff
@@ -213,13 +204,17 @@ public InputStream getInputStream() throws IOException {
 		problemCache.put(uri, problems);
 	}
 
-	private void initializeMavenBuildState() throws ComponentLookupException, InvalidRepositoryException,
-			SettingsParseException, IOException, MavenExecutionRequestPopulationException {
-		if (projectBuilder != null) {
+	private void initializeMavenBuildState(MavenExecutionRequest mavenRequest) {
+		if (projectBuilder != null || mavenRequest == null) {
```

Comment:
should be `mavenRequest != null`

Reply:
@mickaelistria actually.. I'm a bit confused by this. It seems the tests are passing and tests don't pass if I change the if statement to  `if (projectBuilder != null || mavenRequest != null) {` (the tests are passing with how it is currently).

Reply:
Why are you adding `mavenRequest != null`? As far as I understand, the request can totally be non-null while the projectBuilder can still be null.

Reply:
See my other comment. It's the pre-existing code that's confusing me a bit

Reply:
I think the bigger question is: should `initializeMavenbuildState` depend on whether the mavenRequest is null or not?

Reply:
I don't think so, let's try without the condition.

Reply:
Tests seem to pass. Going to merge this.